### PR TITLE
Cluster deployment to AWS region

### PR DESCRIPTION
**Pull request by mesellings**
## Description

Add documentation covering the deployment to automation clusters on AWS as well as GCP.

https://github.com/camunda/product-hub/issues/2393

* Initially just for trial users, further changes may need to be made once rolled out to all users.
* Once initial content is complete, backport from 8.3 upwards.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [X] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [X] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [X] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [X] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by mesellings** [comment]
FYI: @than-pet @boooczek Here is the initial PR - please add to it as required 👍

File: docs/reference/regions.md [hunk 1]
```diff
@@ -26,8 +29,18 @@ Below, find a list of regions currently supported in Camunda 8 SaaS.
...
+
+| AWS region                          | Secondary backups region |
+| :---------------------------------- | :----------------------- |
+| North America, Virginia (us-east-1) | _Not available_          |
+
+To learn more about each region code/location, refer to [AWS regions and availability zones](https://aws.amazon.com/about-aws/global-infrastructure/regions_az/).
 
 :::note
-Running on a Trial or Starter plan and want to try a different region, or interested in other regions or cloud providers? [Contact us](/reference/contact.md) as we are able to make additional regions available on request.
+Only the orchestration cluster is hosted in the AWS region. Management cluster components (Console and Web Modeler) are hosted in the EU on GCP.
```
Comment (than-pet) [inline docs/reference/regions.md:45]: Maybe we could merge this with line 13

File: docs/reference/regions.md [hunk 2]
```diff
@@ -9,15 +9,18 @@ When you create a cluster in Camunda 8 SaaS, you must specify a region for that
 Currently, we make these regions available for customers on the Trial, Starter, and Enterprise Plans. Enterprise customers can discuss custom regions with their Customer Success Manager.
 
 :::note
-Our Console and Web Modeler components are currently hosted in the EU. [Contact us](/reference/contact.md) if you have additional questions.
-:::
 
-Below, find a list of regions currently supported in Camunda 8 SaaS.
+- Console and Web Modeler components are currently hosted in the EU. [Contact us](/reference/contact.md) if you have additional questions.
```
Comment (than-pet) [inline docs/reference/regions.md:13]:
```suggestion
- Console and Web Modeler components are currently hosted in the GCP ( EU ). [Contact us](/reference/contact.md) if you have additional questions.
```
Something like this maybe?

File: docs/reference/regions.md (hunk 1 above)
Comment (mesellings) [reply docs/reference/regions.md]: Agreed - changed 👍

File: docs/reference/regions.md (hunk 2 above)
Comment (mesellings) [reply docs/reference/regions.md]: Agreed - changed 👍

**Comment by mesellings** [comment]
Thanks for the review @than-pet , I made the changes as you suggested 👍

**Review by than-pet — APPROVED** [review]
lgtm

File: docs/reference/regions.md [hunk 3]
```diff
@@ -9,15 +9,18 @@ When you create a cluster in Camunda 8 SaaS, you must specify a region for that
 Currently, we make these regions available for customers on the Trial, Starter, and Enterprise Plans. Enterprise customers can discuss custom regions with their Customer Success Manager.
 
 :::note
-Our Console and Web Modeler components are currently hosted in the EU. [Contact us](/reference/contact.md) if you have additional questions.
-:::
 
-Below, find a list of regions currently supported in Camunda 8 SaaS.
+- Management cluster components (Console and Web Modeler) are currently hosted in GCP (EU). [Contact us](/reference/contact.md) if you have additional questions.
+- Running on a Trial or Starter plan and want to try a different region, or interested in other regions or cloud providers? [Contact us](/reference/contact.md) as we are able to make additional regions available on request.
```
Comment (boooczek) [inline docs/reference/regions.md:14]: Is it intentional to not call out the Enterprise plan here?

Comment (boooczek) [reply docs/reference/regions.md:14]: This part ", or interested in other regions or cloud providers" sounds a bit redundant (asking if you need additional regions) and suggests one might be able to ask for another provider. How about we leave this part out?

File: docs/reference/regions.md [hunk 4]
```diff
@@ -9,15 +9,18 @@ When you create a cluster in Camunda 8 SaaS, you must specify a region for that
 Currently, we make these regions available for customers on the Trial, Starter, and Enterprise Plans. Enterprise customers can discuss custom regions with their Customer Success Manager.
 
 :::note
-Our Console and Web Modeler components are currently hosted in the EU. [Contact us](/reference/contact.md) if you have additional questions.
-:::
 
-Below, find a list of regions currently supported in Camunda 8 SaaS.
+- Management cluster components (Console and Web Modeler) are currently hosted in GCP (EU). [Contact us](/reference/contact.md) if you have additional questions.
```
Comment (boooczek) [inline docs/reference/regions.md:13]: What do you think about calling out here also the fact that the single-tenant clusters can run on a dedicated infrastructure on GCP or AWS? This way it sets the scene for the below list of supported regions.

File: docs/reference/regions.md (hunk 3 above)
Comment (than-pet) [reply docs/reference/regions.md:14]: I think we should remove the "or cloud provider" part.

**Comment by mesellings** [comment]
@boooczek I've made edits for your feedback in the latest commit - please check 👍

**Review by than-pet — DISMISSED** [review]

**Review by boooczek — DISMISSED** [review]
Made one nit comment but apart from that it lgtm so feel free to proceed.

**Comment by mesellings** [comment]
Thanks, @boooczek I made the edit you suggested - this is all good now to release whenever you give me the go-ahead?

**Review by than-pet — APPROVED** [review]

**Pull request merged**